docs/source/topics/proc_configuring-usage-data-collection.adoc

```diff
@@ -0,0 +1,25 @@
+[id="configuring-usage-data-collection_{context}"]
+= Configuring usage data collection
+
+Consent for usage data collection can be granted or revoked by you at any time using the following configuration commands.
+
+.Procedure
+
+* To manually enable telemetry, run the following command:
++
+[subs="+quotes,attributes"]
+----
+$ {bin} config set consent-telemetry yes
+----
+
+* To manually disable telemetry, run the following command:
++
+[subs="+quotes,attributes"]
+----
+$ {bin} config set consent-telemetry no
+----
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about the collected data, see the Red{nbsp}Hat link:https://developers.redhat.com/tool-data-collection[Telemetry data collection notice].
```

Comment:
Here also `Red{nbsp}Hat` and link also need to be updated with https://developers.redhat.com/article/tool-data-collection (current link is detected dead by CI)

Reply:
Thanks, I missed that. I'll also move the URL into the `product-attributes.adoc` file considering we use it in more than one location.